[Dashboard] Remove non functional save button from tag filter panel [ISSUE #158157]

Ruhshan opened the pull request:
## Summary

Fixes #158157

During usage found the save button here has no function. Also from the discussion from the [issue](https://github.com/elastic/kibana/issues/158157), it was found that removal for this save button was expecting behavior. 

<img width="1283" alt="image" src="https://github.com/elastic/kibana/assets/5312918/30d972a2-4770-4194-88eb-0ba1fce2bec0">

Small screen:
<img width="502" alt="image" src="https://github.com/elastic/kibana/assets/5312918/8486b2d3-b1c9-43d6-83ed-6e629de3755c">


### Checklist

Delete any items that are not applicable to this PR.

- [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server))
- [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)


### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

Ruhshan committed:
[Dashboard] Remove non functional save button from tag filter panel

kibanamachine commented:
Since this is a community submitted pull request, a Jenkins build has not been kicked off automatically. Can an Elastic organization member please verify the contents of this patch and then kick off a build manually?

Ruhshan committed:
Merge branch 'main' into dashboard/remove-non-functional-save-button-from-dashboar-tag-filter

doakalexi committed:
[ResponseOps][Alerting] Add missing rule.params descriptions for Stack rules (#161807)

Resolves https://github.com/elastic/kibana/issues/161710

## Summary

Related to this issue: https://github.com/elastic/kibana/issues/161126

Added missing descriptions for the index threshold rule params.


### Checklist

- [x] Any text added follows [EUI's writing
guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses
sentence case text and includes [i18n
support](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/README.md)
- [x] [Unit or functional
tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)
were updated or added to match the most common scenarios

---------

Co-authored-by: lcawl <lcawley@elastic.co>
Co-authored-by: Ying Mao <ying.mao@elastic.co>

szwarckonrad committed:
[Fleet] Kafka integration UI (#160112)

This PR addresses the UI aspect of
https://github.com/elastic/kibana/issues/143324

Happy path


https://github.com/elastic/kibana/assets/29123534/d1664e68-1fb6-42b8-8585-d7132c47d76f

---------

Co-authored-by: kibanamachine <42973632+kibanamachine@users.noreply.github.com>

cnasikas committed:
[Cases] Version `connectors` domain and apis (#162059)

## Summary

This PR versions the `connector` domain object and its corresponding
APIs

### Checklist

Delete any items that are not applicable to this PR.

- [x] [Unit or functional
tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)
were updated or added to match the most common scenarios

### For maintainers

- [x] This was checked for breaking API changes and was [labeled
appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

---------

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

gitstart committed:
[Fleet] Use theme service to display in Dark Mode when set on User Profile (#161819)

Closes https://github.com/elastic/kibana/issues/161135

## What does this PR do?
* Fixes the darkMode theme in fleet page.

## Issue References
* https://github.com/elastic/kibana/issues/161135

## Video/Screenshot Demo 
##### Before

![250689182-c9c4fe55-c007-424a-aedc-b863f1fd0c88-2](https://github.com/elastic/kibana/assets/1501599/71dd9889-3e2d-4b5b-8825-5f7f18206903)

![250689299-34149e80-40b8-493d-a8fc-3079c8cb199c](https://github.com/elastic/kibana/assets/1501599/fdd1b8af-6a5f-4e6c-892e-73baf58d1aab)

###### FIX:

![250690628-2774dc86-487e-489e-b0a2-a1c06539172d](https://github.com/elastic/kibana/assets/1501599/4525bbfa-7771-4611-a099-751c7be54ec6)

![250691578-74d860fa-1db5-47c4-a3b7-425e86720e06](https://github.com/elastic/kibana/assets/1501599/0b03267a-dfb7-48dd-adc3-e9fc3ad5013b)

---
This code was written and reviewed by GitStart Community. Growing future
engineers, one PR at a time.

---------

Co-authored-by: KlingerMatheus <klinger.matheus@gitstart.dev>
Co-authored-by: gitstart_bot <gitstart_bot@users.noreply.github.com>
Co-authored-by: Klinger Matheus <50892465+KlingerMatheus@users.noreply.github.com>
Co-authored-by: Julia Bardi <90178898+juliaElastic@users.noreply.github.com>
Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

kibanamachine committed:
skip failing test suite (#162220)

peteharverson committed:
[ML] Adds a11y tests for categorization and recognizer job wizards (#162126)

## Summary

Adds accessibility tests for the categorization and data recognizer
anomaly detection job wizards.

As part of this PR I have split out the tests for the anomaly detection
pages into a separate file as the original `ml.ts` file was getting very
large.

Part of https://github.com/elastic/kibana/issues/160712
Part of #88496

### Checklist

- [x] [Unit or functional
tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)
were updated or added to match the most common scenarios

shahzad31 committed:
[Synthetics] Simplify journey API query (#162188)

mistic committed:
skip flaky suite (#152206)

sebelga commented:
Hey @Ruhshan,
Thanks for opening this PR. I just commented in the issue you link. We do want to keep the button as it does have a purpose.
If you want to go ahead and change the label from "Save" to "Apply" that would be awesome 👍 
Thanks!

elasticmachine commented:
Pinging @elastic/appex-sharedux (Team:SharedUX)

yngrdyn committed:
[Logs onboarding] Error handling in configureLogs step (#162241)

Closes https://github.com/elastic/kibana/issues/156529.

`Dataset name` is a field that is auto populated based on the `Log file
path`, if the user deletes the autogenerated value they cannot continue
with the next step but before this changes there were no indicative of
why they cannot continue.

### Changes
- Added validation function to datasetName field.
- Added i18n error.

#### Before
<img width="2199" alt="image"
src="https://github.com/elastic/kibana/assets/1313018/24cc166e-c143-4b35-9372-0fcdb4e78f4a">

#### After
<img width="2200" alt="image"
src="https://github.com/elastic/kibana/assets/1313018/72302aa6-1597-439b-8d80-88e7debf1010">

yngrdyn committed:
[Logs onboarding] UI changes in custom logs onboarding flow (#162148)

Relates to https://github.com/elastic/kibana/issues/159655.

### Changes

This PR include the following changes:

#### ConfigureLogs step

- [x]  Change page title into “Stream log files to Elastic”
- [x] Remove stepper
- [x] Remove ’Stream log files to Elastic’ title inside the white box
- [x] Add tooltip to Dataset name with the text “Provide a dataset name
to help identify the source of your logs in future uses. Defaults to the
name of the log file.”
- [x] Add Dataset name input placeholder with “Give your logs a name”
- [x] Change helper text on Dataset name input to “All lowercase, max
100 chars, special characters will be replaced with ‘_’.”
- [x] Add tooltip to Service name with the text “Provide a service name
to allow for distributed services running on multiple hosts to correlate
the related instances.”
- [x] Add Service name input placeholder with “Give your service a name”
- [x] Add tooltip to Namespace (under advanced settings) with the text
“Provide a namespace to customize the grouping of your logs. Defaults to
the default namespace."

##### Before
<img width="1691" alt="image"
src="https://github.com/elastic/kibana/assets/1313018/1b047624-1160-40eb-85ee-199d0874c805">

##### After
<img width="1690" alt="image"
src="https://github.com/elastic/kibana/assets/1313018/572d7615-1363-4892-9b78-994f393a3ebe">


#### InstallShipper step

- [x] Change page title into “Install shipper to collect logs”
- [x] Remove stepper
- [x] Remove ’Install shipper to collect data’ title inside the white
box
- [x] Change first paragraph with “To collect the data from your system
and stream it to Elastic, you first need to install a shipping tool on
the machine generating the logs. In this case, the shipper is an Agent
developed by Elastic.”
- [x] Change text in first step (Install the Elastic agent) with “Select
your platform, and run the install command in your terminal to enroll
and start the Elastic Agent. Do this for each host. Review host
requirements and other installation options before installing.”
- [x] “Host requirements…” should open a link in a new tab to:
[https://www.elastic.co/guide/en/fleet/8.7/elastic-agent-installation.html](https://www.elastic.co/guide/en/fleet/8.7/install-standalone-elastic-agent.html)
- [x] Move up automatic config download switch before the agent download
code block
- [x] Change switch text to “Automatically download the agent’s config”
and add tooltip: “Turn on to add a string to the following code block
that downloads the agent's standard configuration to your host during
installation. Turn off to manually configure the agent in the next
step.”
- [x] When the users turn on the switch, we need to show a callout with
the following text: “Automatically downloading the agent config will
overwrite any existing agent config on your host.”
- [x] During the completion of the first step ‘Install the Elastic
agent’ provide users with callouts that appear as things happen: first
Elastic agent download, then Elastic agent extracted… and so on
- [x] If the user has the switch on (to automatically download the
config), they should also receive the callout confirmation that the
config was written to the yml file
- [x] Change paragraph text of ‘Configure the Elastic agent’ step to
“Add the following configuration to the elastic-agent.yml on the host
where you installed the Elastic agent.”
- [x] Change step 3 titles to “Ship logs to Elastic Observability”
(default state), “Waiting for Logs to be shipped...” (Loading state) and
“Logs are being shipped!” (Success state)
- [x] Change the style of ‘back’ button to Empty + add arrowLeft icon
with primary color
- ~~[ ] Change ‘Inspect’ button with default style, primary color. The
button should become active only when the full installation flow is
finished~~
- ~~[ ] Open the inspect of the installation in a flyout to avoid
interrupting the user flow~~

##### Before
<img width="1674" alt="image"
src="https://github.com/elastic/kibana/assets/1313018/9c0539a3-6cde-4029-a6d3-2978f91b40de">

##### After
<img width="1671" alt="image"
src="https://github.com/elastic/kibana/assets/1313018/84775312-fb5b-4646-90de-6268dec69246">

### Video


https://github.com/elastic/kibana/assets/1313018/e8ae7067-604f-48fd-9b3d-7e7c0791df38

walterra committed:
[ML] AIOps: Rename Explain Log Rate Spikes to Log Rate Analysis. (#161764)

## Summary

Part of #161832.

This PR renames the Explain Log Rate Spikes feature to **Log Rate
Analysis**.

- [x] Renamed references in `docs/developer/*`
- [x] Updated docs screenshots
- [x] Redirect in docs
- [x] Redirect urls from `explain_log_rate_spikes` to `log_rate_analysis`
- [x] API versioning
- [x] Renamed navigation links
- [x] Renamed variable names
- [x] Renamed file names
- [x] Renamed i18n ids
- [x] Renamed breadcrumbs 
- [x] Removed hard coded `AIOPS_ENABLED` feature flag

drewdaemon committed:
[Lens][Event annotations] Move logic into packages (#161500)

tonyghiani committed:
[Logs Explorer]: Update integration icons type (#162219)

## 📓 Summary

The integrations returned from the API have an `icons` property which is
an array of icon object descriptors.

The runtime validation on the clientside was not aligned with the type
of these icons, which are retrieved from the package manifest and have
some optional attributes, while the current implementation assumed all
the properties were mandatory.

This PR aligns the runtime type with the
[PackageSpecIcon](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/common/types/models/package_spec.ts#L114)
type.

<img width="314" alt="Screenshot 2023-07-19 at 11 08 01"
src="https://github.com/elastic/kibana/assets/34506779/ff51bad6-89dd-44f3-a959-24d6717db235">
<img width="315" alt="Screenshot 2023-07-19 at 11 08 18"
src="https://github.com/elastic/kibana/assets/34506779/10cb05bc-c138-4c03-9d8b-74f5a4b7aab0">

Co-authored-by: Marco Antonio Ghiani <marcoantonio.ghiani@elastic.co>

gbamparop committed:
[APM] Fix transaction action menu for trace explorer and dependency operations (#162213)

### Before

https://github.com/elastic/kibana/assets/5831975/3fcceac7-6cf6-4060-972e-86fc4c799115

### After

https://github.com/elastic/kibana/assets/5831975/0eccce82-4ffc-4c9e-8f8f-baf89cfbccec


Closes https://github.com/elastic/kibana/issues/162175

---------

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

mistic committed:
fix(NA): i18n_check

lcawl committed:
[DOCS] Refresh rule action variable screenshots (#162098)

gsoldevila committed:
[FTR - esArchiver] Update the remaining "dynamic" archives, removing mappings.json (#162029)

## Summary

Part of https://github.com/elastic/kibana/issues/161882

- Updates the remaining "dynamic" es archives (aka those using the
`$KIBANA_PACKAGE_VERSION` variable), deleting mappings.json.
- It also removes the piece of logic in `esArchiver` that replaces that
variable.

Ruhshan commented:
> Hey @Ruhshan, Thanks for opening this PR. I just commented in the issue you link. We do want to keep the button as it does have a purpose. If you want to go ahead and change the label from "Save" to "Apply" that would be awesome +1 Thanks!

Sure, on it. Sorry I misunderstood earlier.

jpdjere committed:
[Security Solution] Extend Prebuilt rules install and update workflow test coverage (#161687)

## Summary

- Implement test plan as described in
`x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/installation_and_upgrade.md`

### For maintainers

- [ ] This was checked for breaking API changes and was [labeled
appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

---------

Co-authored-by: kibanamachine <42973632+kibanamachine@users.noreply.github.com>

renovate[bot] committed:
Update dependency xstate to ^4.38.1 (main) (#161056)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

crespocarlos committed:
[Infra UI] Fix theme passed in to charts (#162114)

closes [#110217](https://github.com/elastic/kibana/issues/110217)
## Summary

This PR addresses an issue with how the infra sets the theme in charts.
It replaces the previous implementation with the use of
`useChartsBaseTheme` and `useChartsTheme` provided by the charts
service, ensuring consistent behavior for both light and dark modes.
From a UI perspective, this change doesn't introduce any alterations,
but it solves the problem related to dark mode.

### Screenshots

#### Node Details
<img width="899" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/7ec8b9e9-c440-44d1-86ac-45dbab76a8a9">

<img width="899" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/c6aac4a6-8982-47cc-b08e-ee5d8af0391f">


#### Infra
##### Timeline
<img width="720" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/cd4a1a3c-b5a4-47d5-bbde-5bf54dfac262">

<img width="720" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/839ccaf0-02a5-401e-957a-137d0ef25605">


##### Metrics
<img width="541" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/69138a89-21ff-4f17-bcbd-64ac609dd9e7">

<img width="541" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/63b02a7a-397e-451e-b4d7-7b37fdc4f6a9">


##### Processes
<img width="541" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/01e7d050-0103-4de6-8215-e93b9f80751b">

<img width="541" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/bd1ce95a-1944-411f-abc4-8f2a998f7643">


### Metrics Explorer
<img width="720" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/60d2857a-3b97-4fd6-8cc7-8b7ab757b3d5">

<img width="720" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/98bba0ea-27db-48a2-ae20-ff4162212c6e">


### Alert
#### Inventory rule
<img width="450" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/8b787e53-cb47-4254-ab69-69ab22ff0388">

<img width="450" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/a952bf6b-e22f-4753-8db6-f2deb213b22a">


#### Metric Threshold rule
<img width="450" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/db3d1448-5faa-4e20-bda8-7351d9b52144">

<img width="450" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/a11bc2de-f042-4d4f-9adc-d376205bc23f">


#### Logs rule
<img width="450" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/b703e972-2d47-485e-a2f5-4cfca0c982cd">

<img width="450" alt="image"
src="https://github.com/elastic/kibana/assets/2767137/a98d3827-d930-4ef2-90f9-f336583f15da">




### How to tests
- Set up a local Kibana instance.
- Navigate to the pages mentioned above.
  - Change the user settings to both dark and light mode.
- In light mode, ensure that there are no differences in the charts when
compared to the edge cluster.
- In dark mode, confirm that the issue described in the ticket is
resolved (no horizontal line in white).

---------

Co-authored-by: Kibana Machine <42973632+kibanamachine@users.noreply.github.com>

lcawl committed:
[OAS] Add Elasticsearch query rule parameters (#162001)

paul-tavares committed:
[Security Solution][Endpoint] Unit tests for Policy settings form (#161814)

## Summary

- Adds unit tests for all components of the Policy settings form


### Checklist

- [x] [Unit or functional
tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html)
were updated or added to match the most common scenarios

Ruhshan committed:
[Dashboard] Change tag filter button label

elasticmachine commented:
Pinging @elastic/apm-ui (Team:APM)
